src/couch_replicator_api_wrap.erl

```diff
@@ -83,9 +83,16 @@ db_open(#httpdb{} = Db1, _Options, Create) ->
                     ok
                 end)
         end,
-        send_req(Db, [{method, head}],
-            fun(200, _, _) ->
-                {ok, Db};
+        send_req(Db, [{method, get}],
+            fun(200, _, {Props}) ->
+                case get_value(<<"update_seq">>, Props) of
```

Comment:
Broken checkpoints is just a side effect of missing that field in the response. As like as if `update_seq` will be missed, replication will start and fail with some error in the middle. That's why both are required for clean start.